### ISSUE TITLE
Feature request: Monk run change

### ISSUE THREAD

**Issue by BBalazsJurgen** [Sh0wdown/UnofficialCrusaderPatch#685]
I've just seen the Spearman run change in the pull requests, and instantly loving it, absolutely waiting for the next version to see it in practice in multiplayer. This might just make this good old underused unit a real thing to consider in certain situatinos.

If this is possible however, there is one other unit I would like to ask you to take a look at, and that is the Monk. They suffer from pretty much the same issues as Spearmen, that is being totally devastated by arrows because they are light, but slow. But giving them an "always run" behavior would probably make them way superior to Spearmen.

Do you think it would be possible to make them inherit the behavior of Macemen from the classic, european Stronghold? That is, walk until enemies are within a certain (broad) distance, and run if enemies are (fairly) close? This would make Monks slower to deploy, but still able to provide that much-needed mobility, if necessary.

**Comment by ilosevsrat** [comment]
i think spearman should be fastets non-mounted unit, followed by monk and maceman, if we make monk as fast as spearman it will make recruiting spearman really pointless.

**Comment by BBalazsJurgen** [comment]
Yes, that is exactly the point.
Now, they both always walk (except on aggressive stance).
According to the pull request, spearmen would always run;
--> and so the monk should not always run, but only run if enemies are within range (like the SH Maceman).

I mean, the spearman should definately receive this upgrade yes, but maybe the monk should also get a small bit of help. This way, the relative improvement would be smaller for the monk, and greater for the spearman. The idea is to help the spearman more, but also help the monk a little.

**Comment by GRhin** [comment]
I really think this is a bad idea.
Monks are already the strongest melee unit for their cost, with a pretty decent movement speed (same as pikes). When they run they move faster than the current spearman speed. This is simply too OP. The current balance they are super good at defending within your base, and you can put them on aggressive to run down enemy within 35 tiles (the spearmen work this way too). But allowing them to run more than that makes them a strong, cheap, offensive unit as well as a strong, cheap defensive unit. Which means that you no longer bother with other melee units. 
TLDR; Monks have a specific strength, and you are asking to add them to be strong everywhere instead, which negatively effects other units.

**Comment by LordHansCapon** [comment]
I agree with GRhin, they'd be too strong. I love spearmen, I will be using them all the time when the new release lands. With the ladder bug fix, they are gonna be real good.
